proposals/NNNN-ios-check-type.md

```diff
@@ -0,0 +1,128 @@
+# Explicit returned type from NSDictrionary(Store) category
+
+* Proposal: [SDL-NNNN](NNNN-ios-check-type.md)
+* Author: [Misha Vyrko](https://github.com/mvyrko)
+* Status: **Awaiting review**
+* Impacted Platforms: [iOS]
+
+## Introduction
+
+Specify returned types from NSDictionary (Store). 
```

Comment:
Update: 
**This proposal is to** specify returned types from NSDictionary (Store).